app/uk/gov/hmrc/gform/validation/ValidationResult.scala

```diff
@@ -57,6 +57,10 @@ case class ValidationResult(
   ): FormValidationOutcome = {
 
     val enteredFormFieldValidationResults: List[FormFieldValidationResult] = formFieldValidationResults.collect {
+      case fe @ FieldError(fc @ IsChoice(_), cv, _) =>
+        fe.copy(currentValue =
+          enteredVariadicFormData.userData.many(fc.modelComponentId).getOrElse(Seq.empty[String]).mkString(", ")
+        )
```

Comment:
Can we remove this change? 

It raises a question why `IsRevealingChoice` is not handled here as well.